### ISSUE TITLE
Image/Thumbnail support: Modify thumbnail size support limits for two curated colletions in HDV

### ISSUE THREAD

**Issue by sbarbosadataverse** [IQSS/dataverse.harvard.edu#158]
Two collections require an increase in image file size support to generate thumbnails:
SAEF
Leon Levy

Details will be added here

@cmbz

**Comment by landreev** [comment]
I'm planning to do this outside the Dataverse application - will put together a shell script to select the files that don't have thumbnails, generate the thumbnails and upload them into the S3 bucket. 
We've opened a separate dev. ticket for some in-application mechanism for providing thumbnails for larger files, in the future.

**Comment by landreev** [comment]
These larger images in the SAEF collection, they haven't been uploaded yet, is that correct? 

The ones in the Levy collections I can start processing now. I'll make a script and let it run in the background, in some way that does not compete with the main service for CPU cycles.

**Comment by sbarbosadataverse** [comment]
@cmbz are we ready to have Leonid work on this?

**Comment by landreev** [comment]
Just let me know when you're ready. It's not much work on my end, I'll just start it as a big batch job, and it'll run in the background until done.

**Comment by cmbz** [comment]
@sbarbosadataverse I believe so, certainly re. SAEF. You might confirm with @kmika11 about Leon Levy.

**Comment by kmika11** [comment]
Yes please! One question - will this affect any versioning of the dataset? I suspect not, but just want to confirm.

**Comment by landreev** [comment]
Much belated update - this has been done for SAEF (there are only ~1,500 images larger than 3MB in the collection; with the largest image at 35MB). 
@kmika11 I haven't generated any extra thumbnails for Levy yet, but will start now. And no, it does not generate new versions.

**Comment by sbarbosadataverse** [comment]
@landreev can I close this or do you need it open to work on the Levy images? @kmika11

**Comment by kmika11** [comment]
The thumbnails would be great for Levy, but I would not say that it's a priority.
